docs/templates/index.html

```diff
@@ -99,7 +99,7 @@ <h2>Augmented Markdown</h2>
             {% endblock content %}
         </div>
         <footer>
-            ©2017-2020 — <a class="white" href="http://vincentprouillet.com">Vincent Prouillet</a> and <a class="white" href="https://github.com/getzola/zola/graphs/contributors">contributors</a>
+            ©2017-2019 — <a class="white" href="http://vincentprouillet.com">Vincent Prouillet</a> and <a class="white" href="https://github.com/getzola/zola/graphs/contributors">contributors</a>
```

Comment:
sounds like we went back in time?

Reply:
Wow, I do sometimes forget how Ctrl+A in Vim works when the is `-` in play =\
Thanks for a quick catch!